addons/webhook/certificates.go

```diff
@@ -0,0 +1,203 @@
+package webhook
+
+import (
```

Comment:
knative has a well defined and fairly isolated cert creation utility. Could we leverage https://github.com/knative/pkg/blob/main/webhook/certificates/resources/certs.go#L144 instead of rolling our own?

Reply:
that sounds like a good idea. let me see if it fits all the pieces. It looks like it unless we need the CAkey but I don't think we need it.